### PR TITLE
Add rust-gc reference

### PR DESCRIPTION
**Pull request by jacob-hughes**
This might not be correct, but I've tried to emulate previous @misc entries in the bibfile. So feedback appreciated

### CODE REVIEW

File: softdev.bib [hunk 1]
```diff
@@ -2148,3 +2148,12 @@ @book{gudeman1995representing
     month = Oct,
     publisher= {University of Arizona, Tucson}
 }
+
+ - year is from first git commit https://github.com/chromium/octane
```
Comment (ltratt) [inline softdev.bib]: Huh? Octane?

File: softdev.bib [hunk 2]
```diff
@@ -2148,3 +2148,12 @@ @book{gudeman1995representing
...
+@misc{manish2015rustgc,
+    title = {rust-gc},
+    howpublished = {\href{https://github.com/Manishearth/rust-gc/}{https://github.com/Manishearth/rust-gc/}},
```
Comment (ltratt) [inline softdev.bib]: I think there's a `\link` (or similarly named) macro which avoids the need to repeat information. You might want to check for other web references in the bib file: I think we often also reduce the font size.

File: softdev.bib [hunk 3]
```diff
@@ -2148,3 +2148,12 @@ @book{gudeman1995representing
...
+    note = {Accessed: 2018-08-15},
+    year = {2015},
+    author = {Manishearth},
```
Comment (ltratt) [inline softdev.bib]: Let's use his real name.

File: softdev.bib (hunk 1 above)
Comment (jacob-hughes) [reply softdev.bib]: Oops!

File: softdev.bib (hunk 2 above)
Comment (jacob-hughes) [reply softdev.bib]: Okay, I've used this. Not sure what you mean by reducing font size. This is something you do in the .ltx document ?

File: softdev.bib (hunk 3 above)
Comment (jacob-hughes) [reply softdev.bib]: Done

**Comment by jacob-hughes** [comment]
Ready for squashing I think

**Comment by ltratt** [comment]
Please squash.

**Comment by jacob-hughes** [comment]
Squashed

**Pull request merged**